app/github_oauth.py

```diff
@@ -19,6 +20,7 @@ def generate_github_access_token(github_code):
     github_client_secret = os.getenv("SOCIAL_AUTH_GITHUB_SECRET")
 
     if not github_client_id or not github_client_secret:
+        logging.critical("Github application client id or client secret is missing.")
```

Comment:
Thinking about this, it would be a good idea if the env vars are loaded when the app starts, and errors can be thrown and handled earlier if these critical values are missing